terraform-build-user/main.tf

```diff
@@ -20,6 +20,7 @@ module "iam_user" {
     "/vnc/password",
     "/vnc/ssh/rsa_public_key",
     "/vnc/ssh/rsa_private_key",
+    "/gitlab/personal_authorization_token",
```

Comment:
It's probably not the worst idea for us to alphabetize our list of `ssm_parameters` here.

Reply:
See commit b5d413f.

Reply:
Merci beaucoup!